src/test/feature/ExternalSource/sql/exttab1.sql.source

```diff
@@ -0,0 +1,551 @@
+-- ALTER PROTOCOL demoprot_untrusted OWNER TO extprotu; -- should fail. protocol is not trusted
+-- GRANT SELECT ON PROTOCOL demoprot_untrusted TO extprotu; -- should fail. protocol is not trusted
+-- GRANT SELECT ON PROTOCOL demoprot TO extprotu;
+-- GRANT INSERT ON PROTOCOL demoprot TO extprotu;
+-- SET SESSION AUTHORIZATION extprotu;
+
+-- CREATE WRITABLE EXTERNAL TABLE ext_w(a int) location('demoprot://demoprotfile.txt') format 'text'; -- should succeed
+-- CREATE READABLE EXTERNAL TABLE ext_r(a int) location('demoprot://demoprotfile.txt') format 'text'; -- should succeed
+-- DROP EXTERNAL TABLE IF EXISTS ext_w;
+-- DROP EXTERNAL TABLE IF EXISTS ext_r;
+
+-- RESET SESSION AUTHORIZATION;
+-- REVOKE INSERT ON PROTOCOL demoprot FROM extprotu;
+-- SET SESSION AUTHORIZATION extprotu;
+
+-- CREATE WRITABLE EXTERNAL TABLE ext_w(a int) location('demoprot://demoprotfile.txt') format 'text'; -- should fail
+-- CREATE READABLE EXTERNAL TABLE ext_r(a int) location('demoprot://demoprotfile.txt') format 'text'; -- should succeed
+-- DROP EXTERNAL TABLE ext_r;
+-- RESET SESSION AUTHORIZATION;
+-- REVOKE ALL PRIVILEGES ON PROTOCOL demoprot FROM extprotu;
+-- DROP ROLE IF EXISTS extprotu;
+
+--
+-- WET tests
+--
+
+--
+-- CREATE (including LIKE, DISTRIBUTED BY)
+--
+-- positive
+create writable external table wet_pos1(a text, b text) location('gpfdist://@hostname@:7070/wet.out') format 'text';
+create writable external table wet_pos2(a text, b text) location('gpfdist://@hostname@:7070/wet.out') format 'text' distributed by(b);
+create writable external table wet_pos3(like wet_pos2) location('gpfdist://@hostname@:7070/wet.out') format 'text' distributed by(a,b);
+create writable external table wet_region(like reg_region) location('gpfdist://@hostname@:7070/wet_region.out') format 'text';
+create readable external table ret_region(like wet_region) location('gpfdist://@hostname@:7070/wet_region.out') format 'text';
+
+-- negative
+create writable external table wet_neg1(a text, b text) location('gpfdist://@hostname@:7070@abs_srcdir@/badt1.tbl') format 'text';
+create writable external table wet_neg1(a text, b text) location('gpfdist://@hostname@:7070/wet.out', 'gpfdist://@hostname@:7070/wet.out') format 'text';
+create writable external web table wet_pos5(a text, b text) execute 'some command' on segment 0 format 'text';
+
+--
+-- SELECT from WET (negative)
+--
+select * from wet_pos1;
+
+--
+-- WET: export some data with INSERT SELECT, INSERT and COPY. 
+--
+-- COPY reg_region FROM STDIN DELIMITER '|';
+-- 0|AFRICA|lar deposits. blithely final packages cajole. regular waters are final requests. regular accounts are according to 
+-- 1|AMERICA|hs use ironic, even requests. s
+-- \.
+INSERT INTO wet_region SELECT * from reg_region;
+
+-- COPY wet_region FROM STDIN DELIMITER '|';
+-- 2|ASIA|ges. thinly even pinto beans ca
+-- 3|EUROPE|ly final courts cajole furiously final excuse
+-- \.
+-- INSERT INTO wet_region VALUES(4,'MIDDLE EAST','uickly special');
```

Comment:
The problem is resolved: https://github.com/apache/incubator-hawq/commit/0bc49a345bb97d3d64980164b0fa9e4601c5953d